unaligned_bam_to_bqsr_and_vcf/align.cwl

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: 'Unaligned to aligned BAM'
+requirements:
+    - class: ScatterFeatureRequirement
+    - class: SubworkflowFeatureRequirement
+    - class: MultipleInputFeatureRequirement
+inputs:
+    bam:
+        type: File
+    reference:
+        type: File
+        secondaryFiles: [.fai, .bwt, .sa, .ann, .amb, .pac]
```

Comment:
I think we also want the .alt file. BWA-MEM will use that to avoid discounting mapping score when aligning to alternate haplotypes.